src/act.drive.cpp

```diff
@@ -1996,6 +1996,12 @@ void process_autonav(void)
         return;
       }
       if (veh->in_room == veh->dest) {
+        // QoL - show destination room to vehicle occupants on arrival
+        for (struct char_data *ch = veh->people; ch; ch = ch->next_in_veh) {
+          if (!IS_NPC(ch)) {
```

Comment:
A check here for being flagged screenreader would be good, reader users have definitely asked for the ability to stifle automatic room description delivery.